src/css/_recipe-card-styles.scss

```diff
@@ -0,0 +1,77 @@
+.card-button {
+  border: $border-side-black;
+  height: 27px;
+  margin: 3px;
+}
+
+.add-to-cook-button {
+  background-color: $background-white;
+  display: flex;
+  width: 12%;
+}
+
+.favorite {
+  background-color: $background-white;
+  background-image: url(https://image.flaticon.com/icons/svg/149/149222.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 13%;
+}
+
+.card {
+  background-color: #D7F2FE;
+  border: 3px solid #000;
+  height: 331 px;
+  width: 270px;
+}
+
+.card-header {
+  background-color: $background-white;
+  border-bottom: $border-side-black;
+  display: flex;
+  justify-content: space-between;
+}
+
+.all {
+  grid-gap: 0px;
+  grid-template-columns: repeat(1, 1fr);
+}
```

Comment:
We may have to rename this to say what this property.  All is not a good naming convention.